network/rtp_packet.ksy

```diff
@@ -0,0 +1,85 @@
+meta:
+  id: rtp_packet
+  title: rtp protocol
+  xref:
+    rfc: 3550
+    wikidata: https://en.wikipedia.org/wiki/Real-time_Transport_Protocol
+  license: Unlicense
+  endian: be
+
+doc: |
+  The Real-time Transport Protocol (RTP) is a network protocol for delivering audio and video over IP networks. RTP is used in communication and entertainment systems that involve streaming media, such as telephony, video teleconference applications including WebRTC, television services and web-based push-to-talk features.
+  RTP typically runs over User Datagram Protocol (UDP). RTP is used in conjunction with the RTP Control Protocol (RTCP). While RTP carries the media streams (e.g., audio and video), RTCP is used to monitor transmission statistics and quality of service (QoS) and aids synchronization of multiple streams. RTP is one of the technical foundations of Voice over IP and in this context is often used in conjunction with a signaling protocol such as the Session Initiation Protocol (SIP) which establishes connections across the network.
+  RTP was developed by the Audio-Video Transport Working Group of the Internet Engineering Task Force (IETF) and first published in 1996 as RFC 1889, superseded by RFC 3550 in 2003.
+
+seq:
+  - id: version
+    type: b2
+  - id: padding
+    type: b1
+  - id: extension
+    type: b1
+  - id: csrc_count
+    type: b4
+  - id: marker
+    type: b1
+  - id: payload_type
+    type: b7
+    enum: payload_type_enum
+  - id: sequence_number
+    type: u2
+  - id: timestamp
+    type: u4
+  - id: ssrc
+    type: u4
+  - id: header_extension
+    type: header_extention
+    if: extension==true
+  - id: data
+    size: _io.size-12
```

Comment:
1. Please use spaces in such expressions, i.e. `_io.size - 12`, not `_io.size-12`
2. This seems to just consume everything up to end of stream anyway, why don't just use `size-eos: true`?

Reply:
My point is that right now `size-eos: true` and `size: _io.size - 12` is exactly the same. Even if there is some padding appended after `data`, you're still capturing it in `data` right now.

Reply:
Yes, but I originally wanted to remove the padded data.

Reply:
Take a look, I think I've got it nailed in this commit: d8aaa74a8e66bfab937fc1666ba3d7889b4885a2